### PR TITLE
ci: fix zstd

### PR DESCRIPTION
**Pull request by clementperon**

### CODE REVIEW

**Comment by clementperon** [comment]
@seladb this fix zstd on Windows but now there is an issue with Android...

**Comment by clementperon** [comment]
Related MR on ToyVPN : https://github.com/seladb/ToyVpn-PcapPlusPlus/pull/4

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -443,5 +444,7 @@ jobs:
       working-directory: ./ToyVpn-PcapPlusPlus
       run: |
         sed -i.bak "s|abiFilters.*$|abiFilters '${{ matrix.target }}'|g" app/build.gradle
+        sed -i.bak "s|gradle-6.5-bin|gradle-7.3-bin|g" gradle/wrapper/gradle-wrapper.properties
+        sed -i.bak "s|jcenter()|mavenCentral()|g" build.gradle
```
Comment (seladb) [inline .github/workflows/build_and_test.yml:448]: I already merged https://github.com/seladb/ToyVpn-PcapPlusPlus/pull/4 so I guess this change is no longer needed?

Comment (clementperon) [reply .github/workflows/build_and_test.yml:448]: Correct let me remove it

Comment (clementperon) [reply .github/workflows/build_and_test.yml]: done

**Pull request merged**